nextcord/mentions.py

```diff
@@ -114,20 +100,20 @@ def to_dict(self) -> AllowedMentionsPayload:
         parse = []
         data = {}
 
-        if self.everyone:
+        if self.everyone or self.everyone is None:
```

Comment:
This seems over complicated. Why not check if its None and make it True?

Reply:
It's needed for .merge, which is used throughout the library.

Reply:
I agree that this is over-complicated, but the proposed solution isn't a good idea. Here's a better approach.
```suggestion
        if self.everyone is not False:
```

Reply:
@EmreTech This does not work for checking `self.users`, as that can be List[Snowflake] too